rllab/misc/tensorboard_output.py

```diff
@@ -1,9 +1,14 @@
-import os
+import shutil
+from os.path import dirname, abspath
 
+import google.protobuf.json_format as json_format
 import numpy as np
 import tensorflow as tf
+from jsonmerge import merge
```

Comment:
I think this needs to be alphabetical order?

Reply:
like from imports before imports?
```
from jsonmerge import merge
...
import numpy as np
import tensorflow as tf
```

Reply:
Please make sure all deps are listed in environment.yml

Reply:
Alphabetize imports as if works like "from" or "import" or "as" are not there. only package/module/class/function names matter.